easybuild/easyblocks/h/hypre.py

```diff
@@ -37,25 +38,51 @@
 class EB_Hypre(ConfigureMake):
     """Support for building Hypre."""
 
+    def __init__(self, *args, **kwargs):
+        """Easyblock constructor."""
+
+        super(EB_Hypre, self).__init__(*args, **kwargs)
+
+        self.config_shared = False
+        self.config_static = False
```

Comment:
This flag is mutually exclusive anyway right? So, we can just keep one flag, shared(true) or not(false)? `self.enable_shared = False`

Reply:
Nope, they can be both `True` if built with `configopts = ['', '--enable-shared']`. Then the sanity check is performed for both the static and shared libs.